src/snowflake/snowpark/_internal/ast_utils.py

```diff
@@ -50,6 +50,15 @@
 # Test mode. In test mode, the source filename is ignored.
 SRC_POSITION_TEST_MODE = False
 
+# Debug check for missing AST. This is invoked with various arguments that are expected to be non-NULL if the AST
```

Comment:
Let's move the comment here into the docstring? I.e.
```
def debug_check_missing_ast(ast, container):
   """Debug check..."
   ...
   
```

The parameters are also missing type hints, please add (the precommit should complain about this)

Reply:
the linter GH action doesn't pass yet, can we fix?